src/gmg-server/data/index.js

```diff
@@ -1,14 +1,21 @@
-const SQLite = require('sqlite')
+const {open} = require('sqlite')
+const sqlite3 = require('sqlite3')
 const Path = require('path')
 
 let db
 
 module.exports.initialize = ({ logger }) => {
-   const db_path = Path.join(__dirname, './grill_data.db')
+   const filename = Path.join(__dirname, './grill_data.db')
 
-   logger('Initializing db: [%s]', db_path)
+   logger('Initializing db: [%s]', filename)
 
-   db = SQLite.open(db_path, { Promise })
```

Comment:
This was completely broken. The sqlite upgrade included breaking api changes and the code wasn't updated.